tests/Filter/FilterTest.php

```diff
@@ -117,12 +117,21 @@ public function testOrExpression(string $expected, array $filterOptionsCollectio
 
     public function orExpressionProvider(): iterable
     {
+        $doctrineConfig = $this->createStub(Configuration::class);
```

Comment:
createMock.

`with` doesn't exist for stubs.

Reply:
Done.